File: Beginner/Consumption_1014/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int totalDistance = sc.nextInt();
        double spentFuel = sc.nextDouble();
        double averageConsumption = totalDistance / spentFuel;
        System.out.printf("%.3f km/l\n", averageConsumption);
        sc.close();
    }
    
}

File: Beginner/Consumption_1014/main.go
package main

import (
	"fmt"
)

func main() {
	var distanceTraveled int
	fmt.Scan(&distanceTraveled)
	var spentFuel float64
	fmt.Scan(&spentFuel)

	averageconsumption := float64(distanceTraveled) / (spentFuel)

	fmt.Printf("%.3f km/l\n", averageconsumption)
}
